crates/typst-library/src/foundations/selector.rs

```diff
@@ -266,6 +280,9 @@ impl Repr for Selector {
                     inclusive_arg
                 )
             }
+            Self::Within { selector, ancestor: children_selector } => {
+                eco_format!("{}.contains({})", selector.repr(), children_selector.repr())
```

Comment:
Outdated function name?

```suggestion
                eco_format!("{}.within({})", selector.repr(), children_selector.repr())
```